README.md

```diff
@@ -22,7 +22,7 @@ _Hightouch works to ensure Consent Manager works with most of our product pipeli
 
 ### How it works
 
-1. The consent manager delays loading the events SDK until user consent is given, and will not load the SDK at all if the user opts out of everything. The user's tracking preferences are saved to a cookie and sent as an `identify` trait and a `Consent Updated` track event (if they haven't opted out of everything), ensuring consent data is propagated to your destinations.
```

Comment:
Are we _sure_ this `identify` call here isn't needed? I'm wondering if having it as an identify (plus the track you included) provides benefits for the customer doing data wrangling. It does seem like "do not track me" belongs to a user's identity?

Reply:
I spoke to @kklin about this and he felt it was odd to send both an `identify` and a `Consent Updated` track event. I think whatever we do, it needs to be consistent between `consent-manager` and the newer `consent-tools` (OneTrust) and since the newer `consent-tools` only sends a `Consent Updated` track - I figured we should remove the `identify`. This behavior also matches our [external consent doc](https://www.notion.so/hightouch/External-Consent-management-with-Hightouch-Events-545dff5e208a41209d58010a8283df42?pvs=4#fb5ceaaa3756471daa7702ffac65c0f8).

I'm fine with either case, more just care about consistency between the libraries (which Segment was not).

Reply:
Sounds good. Thanks for providing more of the context. 👍